src/site/stages/build/process-cms-exports/transformers/node-full_width_banner_alert.js

```diff
@@ -0,0 +1,60 @@
+const {
+  getDrupalValue,
+  createMetaTagArray,
+  utcToEpochTime,
+} = require('./helpers');
+
+const transform = entity => ({
+  entityType: 'node',
+  entityBundle: 'full_width_banner_alert',
+  title: getDrupalValue(entity.title),
+  changed: utcToEpochTime(getDrupalValue(entity.changed)),
+  entityMetatags: createMetaTagArray(entity.metatag.value),
+  path: getDrupalValue(entity.path),
```

Comment:
I see that `path` is part of the transformer, so shouldn't it be uncommented in the output schema?

Reply:
I have seen it in the inputs and transformers but not in any of the outputs yet

Reply:
Although, I think the path is used in `entityUrl: { $ref: 'EntityUrl' },` in the ouput

Reply:
So if `path` isn't in the transformed output, then shouldn't it be removed from the transformer?